files/__addonLocation__/babel.config.json

```diff
@@ -1,9 +1,9 @@
 {
 <% if (typescript) { %>  "presets": [["@babel/preset-typescript"]],
-<% } %>  "plugins": [<% if (typescript) { %>
+<% } %>  "plugins": [
+    "@embroider/addon-dev/template-colocation-plugin",<% if (typescript) { %>
     ["@babel/plugin-transform-typescript", { "allowDeclareFields": true }],<% } %>
-    "@embroider/addon-dev/template-colocation-plugin",
-    ["@babel/plugin-proposal-decorators", { "decoratorsBeforeExport": true }],
+    ["@babel/plugin-proposal-decorators", { "legacy": true }],
```

Comment:
Addresses the issue that I described in https://github.com/embroider-build/addon-blueprint/pull/108#discussion_r1210662259.